Content.Shared/_NF/Emp/Components/EmpBlastComponent.cs

```diff
@@ -0,0 +1,30 @@
+using Robust.Shared.GameStates;
+
+namespace Content.Shared._NF.Emp.Components;
+
+/// <summary>
+///     Create circle pulse animation of emp around object.
+///     Drawn on client after creation only once per component lifetime.
+/// </summary>
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState]
+public sealed partial class EmpBlastComponent : Component
+{
+    /// <summary>
+    ///     Timestamp when component was assigned to this entity.
+    /// </summary>
+    [AutoNetworkedField]
+    public TimeSpan StartTime;
+
+    /// <summary>
+    ///     How long will animation play in seconds.
+    ///     Can be overridden by <see cref="Robust.Shared.Spawners.TimedDespawnComponent"/>.
+    /// </summary>
+    [DataField, AutoNetworkedField]
+    public float VisualDuration = 2f;
```

Comment:
this feels really long to me. for a 'blast', the effect lasts a bit too long and makes the screen hard to see if there are more than 1 happening at once.

Reply:
Can try it as 1 sec, the animation speed is fully under control.